nncf/torch/graph/operator_metatypes.py

```diff
@@ -1143,7 +1143,14 @@ def get_operator_metatypes() -> List[Type[OperatorMetatype]]:
 
 OP_NAMES_WITH_WEIGHTS = [x for meta in OPERATORS_WITH_WEIGHTS_METATYPES for x in meta.get_all_aliases()]
 
-QUANTIZE_NODE_TYPES = ["symmetric_quantize", "asymmetric_quantize"]
+QUANTIZE_NODE_TYPES = [
+    "symmetric_quantize",
+    "asymmetric_quantize",
+    "quantize_per_tensor",
+    "dequantize_per_tensor",
+    "quantize_per_channel",
+    "dequantize_per_channel",
```

Comment:
Is it possible to have symmetric/asymmetric + quantize/dequantize + tensor/channel name combinations?